satpy/modifiers/atmosphere.py

```diff
@@ -77,6 +117,13 @@ def __call__(self, projectables, optional_datasets=None, **info):
             refl_cor_band = corrector.get_reflectance(sunz, satz, ssadiff,
                                                       vis.attrs['wavelength'][1],
                                                       red.data)
+        if reduced_correction:
```

Comment:
Is it possible to remove `reduced_correction` and just check if `lim_low`, `lim_high`, and `strength` are all specified? I guess I'm not sure what their defaults would be and which one would be required... :thinking:

Reply:
Then I’ll just set `strength` to 0 as default. No reduction will be applied under it.